crates/testing/tests/tests_1/consensus_task.rs

```diff
@@ -602,3 +602,79 @@ async fn test_vid_disperse_storage_failure() {
 
     run_test![inputs, consensus_script].await;
 }
+
+/// Tests that VID shares that return validation with an Ok(Err) result
+/// are correctly rejected
+#[cfg(test)]
+#[cfg_attr(async_executor_impl = "tokio", tokio::test(flavor = "multi_thread"))]
+#[cfg_attr(async_executor_impl = "async-std", async_std::test)]
+#[cfg(feature = "test-srs")]
```

Comment:
Is it possible to write this test without the feature gate so that it runs in CI? We can run it with the feature gate, but it would take more work and we'd have to add it separately

Reply:
I feature gated it because it relies on changes in Jellyfish that may not be merged for a while.  So I don't think it makes sense to run it in CI until jellyfish is updated.

Reply:
Also, as I suggested above, it may make sense just to merge the fix for now, and wait to merge the test until later.